modules/integration/src/test/resources/broker-auto-start-suite.xml

```diff
@@ -34,17 +34,21 @@
     <!--<parameter name="useDefaultListeners" value="false"/>-->
 
     <test name="broker-integration-tests" verbose="2">
+        <!--<classes>-->
+            <!--<class name="org.wso2.messaging.integration.SuiteInitializer" />-->
+            <!--<class name="org.wso2.messaging.integration.MultipleTopicSubscriberTestCase" />-->
+        <!--</classes>-->
         <packages>
             <package name="org.wso2.messaging.integration"/>
         </packages>
     </test>
 
-    <!--
-    <test name="class-tests" verbose="2">
-        <classes>
-            <class name="org.wso2.messaging.integration.QueueConsumerTest"/>
-        </classes>
-    </test>
-    -->
+
+    <!--<test name="class-tests" verbose="2">-->
+        <!--<classes>-->
+            <!--<class name="org.wso2.messaging.integration.QueueConsumerTest"/>-->
+        <!--</classes>-->
+    <!--</test>-->
```

Comment:
Shall we remove this part since we have it above?

Reply:
Removed